evm-gateway/src/utils.ts

```diff
@@ -22,3 +24,22 @@ export const propsDecoder: PropsDecoder<CFWRequest> = (
     return {};
   }
 };
+
+export const convertIntoMerkleTrieProof = (proof: StateProof) => {
+  const storageProofs: string[] = [];
+  const stateTrieWitness = AbiCoder.defaultAbiCoder().encode(
+    ['bytes[]'],
+    [proof.stateTrieWitness]
+  );
+  for (let index = 0; index < proof.storageProofs.length; index++) {
+    const storageProof = AbiCoder.defaultAbiCoder().encode(
+      ['bytes[]'],
+      [proof.storageProofs[index]]
+    );
+    storageProofs[index] = storageProof;
+  }
+  return {
+    stateTrieWitness,
+    storageProofs,
+  };
```

Comment:
```suggestion
  def flatten(data: string[]) {
    return AbiCoder.defaultAbiCoder().encode(['bytes[]'], [data]);
  }
  return {
    stateTrieWitness: flatten(proof.stateTrieWitness),
    storageProofs: proof.storageProofs.map(flatten),
  };
```

Reply:
Did you mean `function` instead of `def`?